### PR TITLE
chore(console): polish subtitle wording in various pages

### PR DESCRIPTION
**Pull request by charIeszhao**
## Summary

Polish subtitle wording

## Testing

- [x] Locally tested

### CODE REVIEW

File: packages/phrases/src/locales/en/translation/admin-console/applications.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const applications = {
   title: 'Applications',
   subtitle:
-    'Set up a mobile, single page or traditional application to use Logto for authentication',
+    'Set up Logto authentication for your native, single page, M2M, or traditional application',
```
Comment (gao-sun) [inline packages/phrases/src/locales/en/translation/admin-console/applications.ts:4]: machine-to-machine?

**Review by gao-sun — APPROVED** [review]

**Review by xiaoyijun — COMMENTED** [review]
Remenber to add the `// UNTRANSLATED` tag to other locale's related translation.
You can resolve this comment by yourself once done.

**Review by xiaoyijun — COMMENTED** [review]
Remenber to add the `// UNTRANSLATED` tag to other locale's related translation.
You can resolve this comment by yourself once done.

File: packages/phrases/src/locales/en/translation/admin-console/applications.ts (hunk 1 above)
Comment (xiaoyijun) [reply packages/phrases/src/locales/en/translation/admin-console/applications.ts:4]: Remenber to add the `// UNTRANSLATED` tag to other locale's related translation.
You can resolve this comment by yourself once done.

Comment (charIeszhao) [reply packages/phrases/src/locales/en/translation/admin-console/applications.ts:4]: Let's use `machine to machine` and use it in all languages, no neeed to add `// UNTRANSLATED`. Will update soon

**Pull request merged**